server/data/quotes.json

```diff
@@ -526,5 +526,9 @@
   {
     "author": "William Shakespeare",
     "sentence": "My words fly up, my thoughts remain below. Words without thoughts never to heaven go."
+  },
+  {
+    "author": "Winston Churchill",
+    "sentence": "It is a good thing for an uneducated man to read books of quotations."
```

Comment:
Hey there! Please make the quote about **learning a new foreign language** for your PR to be accepted. Thanks!

Reply:
Hi, I've added a new quote by Nelson Mandela.

Removed previous quote.